SwiftUIHW/SwiftUIHW/ContentView.swift

```diff
@@ -7,21 +7,171 @@
+                }
+                createMainView
+                Text("\(isRubbleOrDollar ? totalRubles : totalDolars) \(isRubbleOrDollar ? Constants.rubleCharacter : Constants.dollarCharacter)")
+                    .offset(x: CGFloat(isToggleOn ? Constants.offsetXAnimationValue : Constants.moneyDefaultValue))
+                    .animation(.easeInOut, value: isToggleOn)
+                    .font(.system(size: Constants.walletFontSize))
+            }.confirmationDialog(Constants.emptyStrint, isPresented: $isMoneyAdded) {} message: {
+                Text(Constants.congratilationText)
+            }
+            createChangeValueToggle
+            createToggle
+        }.animation(.easeInOut, value: isToggleOn)
+    }
+    
+    private var createToggle: some View {
+        return Toggle(isOn: $isToggleOn) {
+            Text(Constants.showWalletsText)
+        }.padding()
+         .toggleStyle(.button)
+    }
+    
+    private var createChangeValueToggle: some View {
+        return Toggle(isOn: $isRubbleOrDollar) {
+            Text(Constants.dollarOrRubleText)
+        }.padding()
+    }
+    
+    private var createSendToRublesButton: some View {
+        return Button {
+            isSendRubblesTapped = true
+        } label: {
+            Image(systemName: Constants.arrowUpImageName)
+        }.alert(Constants.typeMoneyValueText, isPresented: $isSendRubblesTapped) {
+            TextField(Constants.typeMoneyValueText, text: $sendingMoneyValue).keyboardType(.numberPad)
+            Button {
+                guard let money = Int(sendingMoneyValue), money <= totalDolars else { return }
+                totalRubles += money * Constants.dollarValue
+                totalDolars = totalDolars - money
+                sendingMoneyValue = Constants.emptyStrint
+            } label: {
+                Text(Constants.sendText)
+            }
+            Button(Constants.cancelText, role: .cancel) {}
+        }
+    }
+    
+    private var createSendToDollarsButton: some View {
+        return Button {
+            isSendDollarsTapped = true
+        } label: {
+            Image(systemName: Constants.arrowDownImageName)
+        }.alert(Constants.typeMoneyValueText, isPresented: $isSendDollarsTapped) {
+            Text(Constants.typeMoneyValueText)
+            TextField(Constants.typeMoneyValueText, text: $sendingMoneyValue).keyboardType(.numberPad)
+            Button {
+                guard let money = Int(sendingMoneyValue), money <= totalRubles else { return }
+                totalDolars += money / Constants.dollarValue
+                totalRubles = totalRubles - money
+                sendingMoneyValue = Constants.emptyStrint
+            } label: {
+                Text(Constants.sendText)
+            }
+            Button(Constants.cancelText, role: .cancel) {}
+        }
+    }
+    
+    private var createMainView: some View {
+        return RoundedRectangle(cornerRadius: Constants.rectangleCornerRadius)
```

Comment:
```suggestion
        RoundedRectangle(cornerRadius: Constants.rectangleCornerRadius)
```
return  не надо писать там, где его не требует явно компилятор